system/lib/libc/README.md

```diff
@@ -6,4 +6,5 @@ Some changes have been made to the version that was taken from upstream, includi
  * Backporting an operator-precedence warning fix from 6e76e1540fc58a418494bf5eb832b556f9c5763e in the upstream version.
  * Switch to using the wasi `fd_write` syscall instead of `writev`.
  * Simplify stdout stream handling: do not support seeking, terminal handling, etc., as it just increases code size and Emscripten doesn't have those features anyhow.
+ * Setting _POSIX_REALTIME_SIGNALS and _POSIX_SPAWN macros to -1, to exclude unsupported functions.
```

Comment:
nit: backticks around the macro names to avoid markdown interpreting the _ as italics.

Reply:
> Are you sure it wouldn't be better to simply not define them? The only places I can find that checks these seems to use "ifndef": https://chromium.googlesource.com/native_client/nacl-newlib/+/master/newlib/libc/sys/linux/sysconf.c#273
> 
> Do you have an example of somewhere these are checked?

[here](https://github.com/videolan/vlc/search?q=_POSIX_SPAWN&unscoped_q=_POSIX_SPAWN) and [there](https://github.com/videolan/vlc/search?q=_POSIX_REALTIME_SIGNALS&unscoped_q=_POSIX_REALTIME_SIGNALS)

Reply:
Sorry I'm not sure what you mean by `#undef at each occurrence`.   Surely we can simply not define it all.  I dont think there is anywhere else in emscripten that we would define there. 

The examples you give seem to define them to -1 if they are undefined, rather than expecting them to be -1 to begin with, so I'm still not totally convinced that the correct behavior is not to simply elide these definitions.    Lets see if we can find some documentation on the these types of macos.

Reply:
I found the docs and it looks like are correct: https://pubs.opengroup.org/onlinepubs/000095399/.  Defining to -1 is the way to go.

Reply:
What I mean is that if the toolchain does not set the macro to the correct value, we have no way of overriding it except if we edit each file.

We thought that since these functions are not implemented in emscripten, they shouldn't be declared as available.